macos/FluentUI/Button.swift

```diff
@@ -253,3 +479,20 @@ open class Button: NSButton {
 
 	private static let pressedBackgroundColorFallbackAlphaComponent:  CGFloat = 0.25
 }
+
+extension NSImage {
+	func tint(color: NSColor) -> NSImage {
+		let image = self.copy() as! NSImage
+		image.lockFocus()
+
+		color.set()
+
+		let imageRect = NSRect(origin: NSZeroPoint, size: image.size)
+		imageRect.fill(using: .sourceAtop)
+
+		image.unlockFocus()
+		image.isTemplate = false
+
+		return image
+	}
+}
```

Comment:
@harrieshin With this, we have tinting of NSImage's on the button working if it is a template image.
Though... maybe we should move this out to it's own file? `NSImage+TintColor.swift` ?

<img width="912" alt="Screen Shot 2020-08-28 at 7 54 10 PM" src="https://user-images.githubusercontent.com/6722175/91627013-e240f580-e968-11ea-8114-d43ea3a3f954.png">

Reply:
the danger with this approach in this scenario is that techincially primary color is different between dark and light mode in iOS and I am going to assume that is true for mac as well. do your image change with system settings change?

Reply:
Good catch. It did not retint with dark mode change. I changed to use contentTintColor, and now it does :D